src/writer.jl

```diff
@@ -313,11 +313,12 @@ end
 
 function render(io::IO, mime::MIME"text/plain", node::Documenter.MarkdownAST.Node, docs::Documenter.DocsNode, page, doc; kwargs...)
     # @infiltrate
+    open_txt = get(page.globals.meta, :CollapsedDocStrings, false) ? "" : "open"
```

Comment:
default is `open` if `meta` not provided, right?, I'm confuse here, shouldn't the `""` be the value from `open_txt` when is available.

Reply:
correct. The conditions are flipped, I agree it is a bit confusing. If `CollapsedDocStrings` is not provided we default to uncollapsed (false/ current default). If it is false then the text is "open" else we give "" which is closed for details block